backend/services/implementations/userService.ts

```diff
@@ -357,11 +376,456 @@ class UserService implements IUserService {
 
         throw error;
       }
-    } catch (error) {
+    } catch (error: any) {
       Logger.error(`Failed to delete user. Reason = ${error.message}`);
       throw error;
     }
   }
+
+  async getVolunteerUserById(userId: string): Promise<VolunteerUserDTO> {
+    let user: User | null;
+    let volunteer: Volunteer | null;
+    let firebaseUser: firebaseAdmin.auth.UserRecord;
+    try {
+      user = await prisma.user.findUnique({
+        where: { id: Number(userId) },
+      });
+
+      volunteer = await prisma.volunteer.findUnique({
+        where: { id: Number(userId) },
+        include: {
+          branches: true,
+          skills: true,
+        },
```

Comment:
```suggestion
        include: {
          branches: true,
          skills: true,
          user: true,
        },
```

Can we include user here? Should avoid doing two separate queries to the two tables, retrieving a volunteer user is a single unit of work

Reply:
As mentioned in [Slack](https://uwblueprint.slack.com/archives/C02K4KBEQVC/p1638844668006500?thread_ts=1638836823.005800&cid=C02K4KBEQVC), let's use `include` wherever applicable :)